site/ja/tutorials/load_data/tf-records.ipynb

```diff
@@ -0,0 +1,1687 @@
+{
```

Comment:
<p>Suggestions:</p><ul><li>"キャッシュする" may be frequently used than "キャッシングする"</li><li>"ファイルで表されます" should be "ファイルで定義されます" because the translation of "define" is "定義"</li><li>Should use same word for the ranslation of "message type". "メッセージの型" and "メッセージタイプ" are appeared.</li><li class="ql-indent-1">I prefer "メッセージ型"</li></ul>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thank you. Adopted your suggestion.</p>